src/styles/helpers/__test__/decomposeColor.spec.js

```diff
@@ -0,0 +1,33 @@
+import { decomposeColor } from '../color';
+
+jest.mock('../color/hexToRgb', () => () => 'rgb(1, 182, 245)');
+
+describe('decomposeColor', () => {
+    const colors = ['#01b6f5', 'rgb(1, 182, 245)', 'rgba(1, 182, 245, 0.5)'];
+    const colorDecomposed = {
+        type: 'rgb',
+        values: [1, 182, 245],
+    };
+
+    it('should return an object', () => {
+        expect.assertions(3);
+        colors.forEach(v => {
+            expect(decomposeColor(v)).toStrictEqual(expect.any(Object));
+        });
+    });
+    it('should return an object with type as string and values as array', () => {
+        expect(decomposeColor(colors[0])).toStrictEqual({
+            type: expect.any(String),
+            values: expect.any(Array),
+        });
+    });
```

Comment:
same ^^
Below when you test ```expect(decomposeColor(colors[0])).toStrictEqual(colorDecomposed);``` that implies that it is an object with type string and values array.